server/services/ProfilesService.js

```diff
@@ -387,27 +375,133 @@ const unblockProfile = function (myId, blockedProfileId) {
   })
 }
 
-const getFollowingsById = function (profileId) {
-  const profileRef = db.collection('users').doc(profileId)
-
-  return profileRef.get().then(doc => {
-    if (!doc.exists) {
-      throw new CustomError({
-        name: 'DatabaseError',
-        message: 'No profile with given id',
-        status: 404
+const getFollowingsById = function(profileId, followingsLimit, paginationId) {
+  const profilesRef = db.collection('users')
+  const followingsQueryRef = db
+    .collection('users')
+    .doc(profileId)
+    .collection('followings')
```

Comment:
If I'm not mistaken this could be rewritten from:

.collection('users')
.doc(profileId)
.collection('followings')

to:
collection(`users/${profileId}/followings`)

Reply:
> If I'm not mistaken this could be rewritten from:
> 
> .collection('users')
> .doc(profileId)
> .collection('followings')
> 
> to:
> collection(`users/${profileId}/followings`)
Done in pull request- Refactor profile backend #71